base/src/main/java/com/mindera/skeletoid/logs/LOG.java

```diff
@@ -348,5 +347,16 @@ public static boolean isInitialized() {
         return mInstance != null;
     }
 
-
+    /**
+     * Sets a custom property of the user
+     *
+     * @param key Property key
+     * @param value Property value
+     */
+    public static void setUserProperty(String key, String value) {
+        if (mInstance == null) {
+            return;
+        }
+        mInstance.setUserProperty(key, value);
```

Comment:
could we perhaps start getting rid of m- prefix in class properties?... wdyt?

Reply:
Migrate, Migrate, Migrate! 🤐

Reply:
👍 we can do it as soon as we convert this to Kotlin 😄